src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,251 @@
+git commit -m "Committing the GlueScript file."
+git push platform master
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> Pushing the script to master won't do anything. Yet. Just a handy place to keep your GlueScript file.
+
+### 3. Enable the integration
+
+Enable the GlueScript integration as follows (you need to have the Platform.sh CLI installed):
+
+```bash
+platform integration:add --type script --file .platform/scripts/my_glue_script.js
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> **note**
+> This is the actual activation of the integration, the script is posted to the API through the CLI. Note that changing the file in Git will not change the integration. To update the integration you will need to use the CLI again.
+>
+> The GlueScript integration can only be managed by the project admin.
+
+# GlueScript APIs
+
+These are the internal Platform.sh APIs that are accessible through GlueScript, API calls are executed with the permissions of the user that created the integration.
+
+## Project API
+
+The project API gives you everything you need to know about the project in the context of which the GlueScript is running. 
+
+For example: `project.environments.master` will give you access to the metadata of the master environment of the current project.
+
+The API starts at `api.project` and is based on a one resource == one object model.
+
+### Properties
+
+The properties of the resource can be access directly on the object, like:
+
+```javascript
+project.owner
+```
+
+### Sub-objects
+
+Sub-objects can be accessed directly on the object, like:
+
+```javascript
+project.settings
+```
+
+### Collections
+
+Resources that are collections support iteration:
+
+```javascript
+project.environments.forEach(function(env) {
+  console.log(env.name);
+});
+```
+
+They also support direct access to an element:
+
+```javascript
+project.environments["master"].name
+```
+
+### Actions
+
+Resource actions are accessible as methods on the object:
+
+```javascript
+project.environments["master"].branch({
+  title="Test",
+  name="test",
+})
+```
+
+## Activity API
+
+The activity API accessed through `activity` gives you all the metadata about the current activity that invoked the GlueScript. The schema is the same as the one for Webhooks, documented in the [Webhook section](https://docs.platform.sh/administration/integrations/webhooks.html) of our documentation.
+
+## Process API
+The process API gives you access to the environment in which the GlueScript is running, such as to environment variables.
+
+For example:
+
+`process.env.FOO` will give you access to the `FOO` environment variable. This is a great place to access API keys, or other secrets.
+
+## Storage API
+
+A storage API is offered for integrations to store information between calls. E.g. if you want to be able to edit a previous slack message. The storage is scoped per integration.
```

Comment:
We haven't made it clear what defines an "integration." Does one script run by the GlueScript integration get its own storage scope?

Also, what happens if I push an updated script. Is that a new integration, or does it retain its storage scope?

Reply:
1. An integration is a "script".
2. I don't even know we can update it (please verify). Probably if we can, and the ID stays stable it will. A good thing to test.